pkg/vault/contracts/AggregatorRouter.sol

```diff
@@ -32,6 +32,11 @@ contract AggregatorRouter is IAggregatorRouter, SenderGuard, VaultGuard, Reentra
         // solhint-disable-previous-line no-empty-blocks
     }
 
+    /// @inheritdoc IAggregatorRouter
+    function getVault() public view returns (IVault) {
```

Comment:
I guess it's public vs. external so that it can be used in descendant contracts?